analog-weather-glow/usr/share/asteroid-launcher/watchfaces/analog-weather-glow.qml

```diff
@@ -352,7 +368,7 @@ Item {
             anchors.fill: parent
             opacity: hrmSensorActive ? activeArcOpacity : inactiveArcOpacity
             smooth: true
-            renderStrategy : Canvas.Cooperative
+            renderStrategy: Canvas.NoPaint // Only redraw when explicitly requested
```

Comment:
This is suspect -- there is no "Canvas.NoPaint" in the documentation: https://doc.qt.io/qt-5/qml-qtquick-canvas.html#renderStrategy-prop